File: platform/sdk/nodejs/examples/cloud/cloud-021_quota_retry_after_header.js
const { CloudClient } = require("../../xuanji-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
const result = client.quotaRetryAfterHeader(150);
if (!result.throttled || result.retryAfterSeconds <= 0) process.exit(1);
console.log("XJ-OK: cloud-021_quota_retry_after_header");
process.exit(0);

File: platform/sdk/python/examples/cloud/cloud-021_quota_retry_after_header.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from xuanji_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
result = client.quotaRetryAfterHeader(150)
if not result["throttled"] or result["retryAfterSeconds"] <= 0:
    sys.exit(1)
print("XJ-OK: cloud-021_quota_retry_after_header")
sys.exit(0)
